### DISCUSSION TITLE
Certificate generation failing because of illegal chars in Course name

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-Knowlg/Community#73] [General Discussions]
Certificate generation was seen to fail because of bad characters in the Course name. The back end addresses this by dropping such chars from the Course name when the certificate is being generated. 
Ideally, the front end should restrict the user from entering such chars into the Course name, as it may lead to an inconsistency between the names on the Course in the system vs. what is on the certificate - if not done.
Is there a way to ensure both the BBs (Knowlg and Lern) work off the same set of configurable chars so that there is no inconsistency/ the list need not be duplicated?
@vinukumar-vs [reshmi-nair](https://github.com/reshmi-nair)  @maheshkumargangula

**Comment by vinukumar-vs** [comment]
@rhwarrier We already have a related bug in the Knowlg issue tracker. Here is the link
https://project-sunbird.atlassian.net/browse/KN-548

We can track the status in the above bug.

@AmiableAnil
